### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by evilchia**
updating dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,10 +9,9 @@
 		"package": "npx lerna run package",
 		"install": "npx lerna exec -- npm install"
 	},
-	"dependencies": {},
 	"devDependencies": {
-		"lerna": "8.1.2",
-		"rimraf": "5.0.5"
+		"lerna": "^8.1.8",
```
Comment (matthewwerny) [inline package.json:13]: These versions aren't fixed and will likely cause unexpected breakages as versions become available and shift.

File: packages/build/bin/Utils.js [hunk 2]
```diff
@@ -3,24 +3,25 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
     return (mod && mod.__esModule) ? mod : { "default": mod };
 };
 Object.defineProperty(exports, "__esModule", { value: true });
-exports.loadJson = exports.overlay = exports.merge = exports.isDefined = exports.requireNotNull = void 0;
+exports.requireNotNull = requireNotNull;
```
Comment (matthewwerny) [inline packages/build/bin/Utils.js:6]: Anything in this bin folder is derived from the Typescript sources and doesn't need to be edited manually as the next build will overwrite it.

File: packages/build/package.json [hunk 3]
```diff
@@ -29,18 +29,18 @@
...
-		"@jest/globals": "^28.1.1",
-		"@types/jest": "28.1.3",
-		"express": "4.17.2",
-		"fs-extra": "10.0.0",
-		"jest-cli": "^28.1.1",
-		"jest-environment-jsdom": "^28.1.1",
-		"ts-jest": "^28.0.5",
-		"typescript": "4.7.4"
+		"@babel/preset-env": "^7.25.4",
+		"@babel/preset-typescript": "^7.24.7",
```
Comment (matthewwerny) [inline packages/build/package.json:33]: These versions aren't fixed and will likely cause unexpected breakages as versions become available and shift.

File: packages/integration/package.json [hunk 4]
```diff
@@ -65,39 +65,39 @@
...
-		"@testing-library/dom": "9.3.4",
-		"@testing-library/jest-dom": "6.3.0",
-		"@types/jest": "29.5.11",
-		"@types/jsdom": "21.1.6",
-		"@types/lodash": "4.14.202",
-		"@types/node": "20.11.6",
-		"@typescript-eslint/eslint-plugin": "7.3.1",
-		"@typescript-eslint/parser": "7.5.0",
+		"@babel/cli": "^7.25.6",
+		"@babel/core": "^7.25.2",
```
Comment (matthewwerny) [inline packages/integration/package.json:69]: These versions aren't fixed and will likely cause unexpected breakages as versions become available and shift.

File: packages/sandbox/package.json [hunk 5]
```diff
@@ -22,35 +22,35 @@
 		}
 	},
 	"devDependencies": {
-		"@babel/core": "7.21.3",
+		"@babel/core": "^7.25.2",
```
Comment (matthewwerny) [inline packages/sandbox/package.json:25]: These versions aren't fixed and will likely cause unexpected breakages as versions become available and shift.

File: packages/schema/package.json [hunk 6]
```diff
@@ -20,6 +20,6 @@
   },
   "homepage": "https://github.com/cydran/cydran#readme",
   "devDependencies": {
-    "ajv": "^8.11.0"
```
Comment (matthewwerny) [inline packages/schema/package.json:23]: These versions aren't fixed and will likely cause unexpected breakages as versions become available and shift.

File: packages/testsupport/package.json [hunk 7]
```diff
@@ -63,47 +63,47 @@
...
-		"@babel/core": "7.23.7",
-		"@babel/plugin-transform-modules-umd": "7.23.3",
-		"@babel/preset-env": "7.23.8",
-		"@babel/preset-typescript": "7.23.3",
-		"@types/lodash": "4.14.202",
-		"@types/node": "20.11.6",
-		"@typescript-eslint/eslint-plugin": "7.3.1",
-		"@typescript-eslint/parser": "7.5.0",
+		"@babel/cli": "^7.25.6",
+		"@babel/core": "^7.25.2",
```
Comment (matthewwerny) [inline packages/testsupport/package.json:67]: These versions aren't fixed and will likely cause unexpected breakages as versions become available and shift.

**Review by matthewwerny — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (evilchia) [reply package.json:13]: fixing

File: packages/build/bin/Utils.js (hunk 2 above)
Comment (evilchia) [reply packages/build/bin/Utils.js:6]: should not be part of scm then...removed

File: packages/build/package.json (hunk 3 above)
Comment (evilchia) [reply packages/build/package.json:33]: fixing

File: packages/integration/package.json (hunk 4 above)
Comment (evilchia) [reply packages/integration/package.json:69]: fixing

File: packages/sandbox/package.json (hunk 5 above)
Comment (evilchia) [reply packages/sandbox/package.json:25]: fixing

File: packages/schema/package.json (hunk 6 above)
Comment (evilchia) [reply packages/schema/package.json:23]: fixing

File: packages/testsupport/package.json (hunk 7 above)
Comment (evilchia) [reply packages/testsupport/package.json:67]: fixing

**Review by matthewwerny — APPROVED** [review]

**Pull request merged**